<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorRange</class>
 <widget class="QWidget" name="InspectorRange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="elementName">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Range</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>2</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QToolButton" name="resetNoteHeadGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>reset value</string>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/resetproperty.png</normaloff>:/data/icons/resetproperty.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>10</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Head type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Head group</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Direction</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QToolButton" name="resetLineWidth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>reset value</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/resetproperty.png</normaloff>:/data/icons/resetproperty.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>10</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Top note</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="direction">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Upright</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Leaning Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Leaning Right</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="topTpc">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maxVisibleItems">
          <number>15</number>
         </property>
         <property name="maxCount">
          <number>40</number>
         </property>
         <item>
          <property name="text">
           <string>[undefined]</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♯♯</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Oct </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="topOctave">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="bottomTpc">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maxVisibleItems">
          <number>15</number>
         </property>
         <property name="maxCount">
          <number>40</number>
         </property>
         <item>
          <property name="text">
           <string>[undefined]</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A♯♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♭♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♭</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♯</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B♯♯</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Oct </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="bottomOctave">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Bottom note</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="hasLine">
       <property name="text">
        <string>Has line</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="noteHeadType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Whole</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Half</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quarter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Breve</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="lineWidth">
       <property name="suffix">
        <string>sp</string>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="resetHasLine">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>reset value</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/resetproperty.png</normaloff>:/data/icons/resetproperty.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>10</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="resetNoteHeadType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>reset value</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/resetproperty.png</normaloff>:/data/icons/resetproperty.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>10</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="resetDirection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>reset value</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/resetproperty.png</normaloff>:/data/icons/resetproperty.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>10</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="noteHeadGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>normal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cross</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>diamond</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>triangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>slash</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>xcircle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>do</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>re</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>fa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sol</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>la</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ti</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>alt. brevis</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Line width</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>noteHeadGroup</tabstop>
  <tabstop>resetNoteHeadGroup</tabstop>
  <tabstop>noteHeadType</tabstop>
  <tabstop>resetNoteHeadType</tabstop>
  <tabstop>direction</tabstop>
  <tabstop>resetDirection</tabstop>
  <tabstop>hasLine</tabstop>
  <tabstop>resetHasLine</tabstop>
  <tabstop>lineWidth</tabstop>
  <tabstop>resetLineWidth</tabstop>
  <tabstop>topTpc</tabstop>
  <tabstop>topOctave</tabstop>
  <tabstop>bottomTpc</tabstop>
  <tabstop>bottomOctave</tabstop>
 </tabstops>
 <resources>
  <include location="../musescore.qrc"/>
 </resources>
 <connections/>
</ui>
